compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -178,12 +179,80 @@ trait ParallelTesting extends RunnerOrchestration { self =>
         .getOrElse("")
       }
       .toList.sortBy(_._1).map(_._2.filter(isSourceFile).sorted)
+
+    def sourceFiles: Array[JFile] = compilationGroups.flatten.filter(isSourceFile).toArray
+  }
+
+  private trait CompilationLogic { this: Test =>
+    val suppressErrors = false
+
+    final def compileTestSource(testSource: TestSource): Either[Throwable, List[TestReporter]] =
+      Try(testSource match {
+        case testSource @ JointCompilationSource(name, files, flags, outDir, fromTasty, decompilation) =>
+          val reporter =
+            if (fromTasty) compileFromTasty(               flags, suppressErrors, outDir)
+            else           compile(testSource.sourceFiles, flags, suppressErrors, outDir)
```

Comment:
We almost never use column alignment in the Dotty codebase. You should remove the extra spaces.